Formula/libggz.rb

```diff
@@ -1,6 +1,6 @@
 class Libggz < Formula
   desc "The libggz library wraps many common low-level functions"
-  homepage "http://dev.ggzgamingzone.org/libraries/libggz/"
+  homepage "https://web.archive.org/web/20180511124057/dev.ggzgamingzone.org/libraries/libggz/"
```

Comment:
This software is abandoned, not maintained, it has 2 installs in 90 days, let's remove the formula.